recipes/mlem/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "mlem" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9ab945158ca5ecd772744a50d6d8d871cc7e11e140319ea8ed93a5adc17bf5e9
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - mlem = mlem.cli:cli
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - dill
+    - requests
+    - isort
+    - docker
+    - pydantic <2
+    - click <9
+    - aiohttp <4
+    - aiohttp_swagger <2
+    - Jinja2 <3
+    - fsspec >=2021.7.0
+    - fusepy
+    - cached-property
+    - entrypoints
+    - setuptools
```

Comment:
These are optional requirements. In normal situation, MLEM doesn't aim to install most of those. It expects that user will have some of them installed and we just use it. E.g., if user doesn't have tensorflow installed, everything will work fine, because he won't have models trained in tensorflow in the first place. With all that said, do we need to list these?